foundry.toml

```diff
@@ -6,8 +6,6 @@ optimizer = true
 optimizer_runs = 999999
 solc_version = "0.8.15"
 evm_version = "cancun"
-via_ir = true
-broadcast = 'records'
```

Comment:
These foundry config changes are not directly related to the rehearsal, and are just general simplifications we are making here